src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -49,13 +60,43 @@ public void setArmGoal(double targetPosition, double targetVelocity) {
       targetPosition = getArmClampedGoal(targetPosition);
       
     }
-    
+
+    public double getArmPos() {
+      return MathUtil.inputModulus(armEncoder.getPosition(), Constants.Arm.ARM_DICONT_RAD,
+              Constants.Arm.ARM_DICONT_RAD + 2 * Math.PI);
+  }
+
+    public double getKg() {
```

Comment:
not needed.
only need to dynamically calculate kG if two-jointed arm, but this arm has only 1 joint.